startup.sh

```diff
@@ -32,12 +32,31 @@ function handleStartup() {
     if [ ! -z "$ADMINPASS" ] && [ ! -a "$ADMINMAIL" ]; then
       /opt/kimai/bin/console kimai:create-user superadmin $ADMINMAIL ROLE_SUPER_ADMIN $ADMINPASS
     fi
+    # I think kimai:install overwrites the monolog config
+    if [ "$APP_ENV" == "dev" ]; then
+      cat <<EOF > /opt/kimai/config/packages/dev/monolog.yaml
+monolog:
+  handlers:
+    main:
+      type: error_log
+      level: debug
+EOF
+    else 
+      cat <<EOF > /opt/kimai/config/packages/prod/monolog.yaml
+monolog:
```

Comment:
this config looks different than the [default kimai monolog config](https://github.com/kevinpapst/kimai2/blob/master/config/packages/prod/monolog.yaml) - on purpose?

Reply:
Yes, that's the fix. It redirects the output the server log so that it runs in the foreground and dumps to stdout.  That way the logs are visible when running the docker.